tests/utils/LinearAlg2DTest.cpp

```diff
@@ -363,5 +363,12 @@ void LinearAlg2DTest::getPointOnLineWithDistAssert(const Point p, const Point a,
     CPPUNIT_ASSERT_MESSAGE(ss.str(), (!actual_returned && !supposed_returned) || (actual_returned && vSize2(actual_result - supposed_result) < 10 * 10 && std::abs(returned_dist - dist) < 10));
 }
 
+void LinearAlg2DTest::rotateAroundTest()
+{
+    Point3Matrix mat = LinearAlg2D::rotateAround(Point(10, 17), 90);
```

Comment:
How about a test for negative rotation, zero rotation, or something that's not a multiple of 90?

Reply:
Good idea, but for me it's going to take too much time.

Perhaps you can make a JIRA ticket?

Reply:
I've made a few tests now :smiley:

Reply:
Well I am busy, and I do like it if you fix all the problems yourself rather than me having to spend time on it, but that wouldn't be fair.

I think my PRs should be handled the same as external ones.